packages/twenty-front/src/modules/activities/files/components/AttachmentRow.tsx

```diff
@@ -70,17 +73,63 @@ export const AttachmentRow = ({ attachment }: { attachment: Attachment }) => {
     deleteOneAttachment(attachment.id);
   };
 
+  const { updateOneRecord: updateOneAttachment } = useUpdateOneRecord({
+    objectNameSingular: CoreObjectNameSingular.Attachment,
+  });
+
+  const handleUpdate = () => {
+    updateOneAttachment({
+      idToUpdate: attachment.id,
+      updateOneRecordInput: { name: newName },
+    });
+    setIsEditing(false);
+  };
+
+  const handleRename = () => {
+    setIsEditing(true);
+  };
+
+  const handleInputChange = (text: string) => {
+    setNewName(text);
+  };
+
+  const handleInputBlur = () => {
+    handleUpdate();
+  };
+  useEffect(() => {
```

Comment:
This logic should be inside the EditableField component

Reply:
Done.